explainer.md

```diff
@@ -263,6 +266,22 @@ any URL under their scope.
 Documents can only obtain the cookies at their current URL. In other words,
 the only valid `url` value in Document contexts is the document's URL.
 
+### Read a partitioned cookie
+
+If the user agent supports
+[cookie partitioning](https://github.com/DCtheTall/CHIPS), then the cookie
+objects will have a boolean value indicating if the cookie is partitioned.
+
+```javascript
+// Read a cookie set without the Partitioned attribute.
+const cookie = await cookieStore.get('session_id');
+console.log(cookie.partitioned);  // -> false
+
+// Read a Partitioned cookie from a third-party context.
```

Comment:
CHIPS [does not mandate](https://github.com/privacycg/CHIPS#handling-older-or-incompatible-clients) use of the __Host-prefix, 

>Although it is not required, it is still recommended to still include the __Host- prefix. Even clients that do not recognize the Partitioned attribute still enforce the semantics of the __Host- prefix. 

so a request could present two cookies with the same name. Will `cookieStore.get()` fail or simply return the first of the two cookies? In that situation, would `cookieStore.getAll()` retrieve the two cookies?

Reply:
#221 is relevant here - there are existing situations where multiple cookies with the same name can be present, not a new situation with partitioned cookies.

Currently as spec'd and implemented in Chrome, the first cookie is returned.